utils/tuple_utils.hh

```diff
@@ -11,6 +11,8 @@
 
 #include <functional>
 #include <tuple>
+#include <seastar/core/shared_ptr.hh>
+#include <seastar/core/sharded.hh>
 
```

Comment:
Please move the new code to a new file rpc-utils.hh. The new code isn't about tuples, it's about rpc, and random code that wants to manipulate tuples shouldn't learn about shared_ptr etc.

Reply:
done